WordPress/Classes/Extensions/NSAttributedStringKey+Conversion.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+import UIKit
+
+
+// MARK: - NSAttributedStringKey Helpers
+//
+extension NSAttributedStringKey {
```

Comment:
I found it a bit odd that this was an extension on `NSAttributedStringKey` when it's about converting dictionaries.
I often like to see if there's a generic problem being solved, and I think this can be stated as converting a dictionary with keys that are `RawRepresentable`

What do you think about this?
```swift
extension Dictionary where Key: RawRepresentable, Key.RawValue: Hashable {
    init(rawKeyedDictionary: Dictionary<Key.RawValue, Value>) {
        self.init(uniqueKeysWithValues: rawKeyedDictionary.lazy.flatMap({ (rawKey, value) in
            guard let key = Key(rawValue: rawKey) else {
                return nil
            }
            return (key, value)
        }))
    }

    var rawKeyed: [Key.RawValue: Value] {
        return Dictionary<Key.RawValue, Value>(uniqueKeysWithValues: map({ ($0.key.rawValue, $0.value)}))
    }
}

// Example
let attributes: [NSAttributedStringKey: Any] = [
    .strokeWidth : 1,
]
let rawAttributes = attributes.rawKeyed
[NSAttributedStringKey: Any](rawKeyedDictionary: rawAttributes)
```

Reply:
+1 to what Diego said. Although i do love the generic implementation, this is a temporary workaround (we've got the exact same thing in Aztec). (And it's a bit easier to figure out what's going on as well!)